### PR TITLE
Stat info merge

### PR DESCRIPTION
**Pull request by alisonrag**
tested in: servertype 20170614
image

wait some tests

iRO
image

bRO
image

tested in RMS
image

### CODE REVIEW

File: src/Network/Receive.pm [hunk 1]
```diff
@@ -435,6 +645,427 @@ sub connection_refused {
...
+	my $actor = {
+		'00B0' => $char,
+		'00B1' => $char,
+		'00BE' => $char,
+		'0141' => $char,
+		'01AB' => exists $args->{ID} && Actor::get($args->{ID}),
+		'02A2' => $char->{mercenary},
+		'07DB' => $char->{homunculus},
+		'0ACB' => $char,
+		'081E' => exists $args->{ID} && Actor::get($args->{ID}), # Sorcerer's Spirit
```
Comment (sctnightcore) [inline src/Network/Receive.pm]: Maybe $char->{Elemental}, ?

File: src/Network/Receive.pm [hunk 2]
```diff
@@ -525,7 +1156,7 @@ sub actor_display {
 				NPC_PET_TYPE, 'Actor::Pet',
 				NPC_HO_TYPE, 'Actor::Slave',
 				NPC_MERSOL_TYPE, 'Actor::Slave',
-				# NPC_ELEMENTAL_TYPE? # not encountered
+				NPC_ELEMENTAL_TYPE, 'Actor::Elemental', # Sorcerer's Spirit
```
Comment (sctnightcore) [inline src/Network/Receive.pm]: Miss Elemental.pm in src\Actor

File: src/Network/Receive.pm (hunk 1 above)
Comment (alisonrag) [reply src/Network/Receive.pm]: can be other elemental fom another socerer

File: src/Network/Receive.pm (hunk 2 above)
Comment (alisonrag) [reply src/Network/Receive.pm]: fixed

File: src/Network/Receive.pm [hunk 3]
```diff
@@ -357,6 +357,47 @@ use constant {
...
 
+
+use constant {
+	EXP_FROM_BATTLE => 0x0,
+	EXP_FROM_QUEST => 0x1,
+};
+
+# 07F6 (exp) doesn't change any exp information because 00B1 (exp_zeny_info) is always sent with it
+# r7643 - copy-pasted to RagexeRE_2009_10_27a.pm
+sub exp {
```
Comment (sctnightcore) [inline src/Network/Receive.pm]: src\Network\Receive\kRO\RagexeRE_2009_10_27a.pm
image

Comment (alisonrag) [reply src/Network/Receive.pm]: removed duplicated code, thanks

**Pull request merged**